### ISSUE TITLE
QNN support

### ISSUE THREAD

**Issue by MeghnaNatraj** [lutzroeder/netron#1395]
This feature was requested before. Refer to  #1283 (**This feature is still missing in Netron**)

TL;DR: We want to visualize the QNN model (serialized.bin file). Currently, I get the error "File contains undocumented QNN serialized context." when I try to open the file in Netron. Is it possible to add support to visualize the this file?

Reference:
There are 4 files generated when converting a model from TFLite to QNN. Let us take the Resnet model as an example. You can refer to the files here ([TFLite and QNN models](https://drive.google.com/drive/folders/1zAwPDfzHe2_II2x_pLIqUCzAjm0LNiG3?usp=sharing)):

TFLite:
1. resnet.tflite: Can be visualized on Netron

QNN files:
1. **[resnet.serialized.bin](https://drive.google.com/file/d/1J7JRSnQ0-aYrlCrk7EDfCOatLFhQrhmU/view?usp=drive_link) (actual model): Cannot run on Netron**
2. resnet.json: (not the model, just a JSON with a general model structure, fewer ops, etc): Can be visualized on Netron
3. resnet.cpp: (not relevant)
4. resnet.bin: (not relevant)

**Comment by lutzroeder** [comment]
Duplicate of #1283
